encord/orm/ontology.py

```diff
@@ -0,0 +1,170 @@
+    ADMIN = 0
+    USER = 1
+
+
+class Ontology(dict, Formatter):
+    def __init__(
+        self,
+        title: str,
+        structure: OntologyStructure,
+        ontology_hash: str,
+        description: Optional[str] = None,
+    ):
+        """
+        DEPRECATED - prefer using the :class:`encord.ontology.Ontology` class instead.
+
+        This class has dict-style accessors for backwards compatibility.
+        Clients who are using this class for the first time are encouraged to use the property accessors and setters
+        instead of the underlying dictionary.
+        The mixed use of the `dict` style member functions and the property accessors and setters is discouraged.
+
+        WARNING: Do NOT use the `.data` member of this class. Its usage could corrupt the correctness of the
+        datastructure.
+        """
+        super().__init__(
+            {
+                "ontology_hash": ontology_hash,
+                "title": title,
+                "description": description,
+                "structure": structure,
+            }
+        )
+
+    @property
+    def ontology_hash(self) -> str:
+        return self["ontology_hash"]
+
+    @property
+    def title(self) -> str:
+        return self["title"]
+
+    @title.setter
+    def title(self, value: str) -> None:
+        self["title"] = value
+
+    @property
+    def description(self) -> str:
+        return self["description"]
+
+    @description.setter
+    def description(self, value: str) -> None:
+        self["description"] = value
+
+    @property
+    def structure(self) -> OntologyStructure:
+        return self["structure"]
+
+    @structure.setter
+    def structure(self, value: OntologyStructure) -> None:
+        self["structure"] = value
+
+    @classmethod
+    def from_dict(cls, json_dict: Dict) -> Ontology:
+        return Ontology(
+            title=json_dict["title"],
+            description=json_dict["description"],
+            ontology_hash=json_dict["ontology_hash"],
+            structure=OntologyStructure.from_dict(json_dict["editor"]),
+        )
+
+
+class CreateOntologyResponse(dict, Formatter):
+    def __init__(
+        self,
+        title: str,
+        storage_location: int,
+        ontology_hash: str,
+        user_hash: str,
+    ):
+        """
+        This class has dict-style accessors for backwards compatibility.
+        Clients who are using this class for the first time are encouraged to use the property accessors and setters
+        instead of the underlying dictionary.
+        The mixed use of the `dict` style member functions and the property accessors and setters is discouraged.
+
+        WARNING: Do NOT use the `.data` member of this class. Its usage could corrupt the correctness of the
+        datastructure.
+        """
+
+        super().__init__(
+            {
+                "title": title,
+                "type": storage_location,
+                "ontology_hash": dataset_hash,
```

Comment:
a bit of a blunder. Using PyCharm or a Python linter will help

Reply:
Thanks!

My `pylint` setup relies on having settings in `pyproject.toml`, so disables itself for the SDK. Will need to fix that.